lib/reruby/git_autocommit.rb

```diff
@@ -0,0 +1,25 @@
+require 'git'
+
+module Reruby
+  class GitAutocommit
+    AUTOCOMMIT_MSG = 'Reruby autocommit before refactoring'.freeze
+    ROOT_PATH = File.expand_path('../../', File.dirname(__FILE__))
```

Comment:
I think this one is problematic. It's getting the root for the gem (something like /var/ruby/gems/reruby), not the path for the project you are refactoring. The way the code searcher works assumes that reruby is being run from the root of the project, so using $PWD should work better.